examples/simple.py

```diff
@@ -0,0 +1,11 @@
+import board
+import time
+from opt3001 import OPT3001
+
+i2c = board.I2C()
+
+opt = OPT3001(i2c)
```

Comment:
I like this, maybe add a comment mentioning the default i2c address and how it can be configured - perhaps add the comment from the driver that mentions what address corresponds to which connection